packages/ui/src/theme/tokens/types/designToken.ts

```diff
@@ -18,6 +18,11 @@ export function isDesignToken(arg: unknown): arg is WebDesignToken {
  */
 export type DesignToken<ValueType = any> = {
   value: ValueType;
+  /**
+   * The deprecatedStateToken flag will mark tokens that should be filtered out of the theme variable output
+   * This is being used to create duplicate state entries for those states that are not conforming to our current naming convention i.e. hover vs _hover
+   */
+  deprecatedStateToken?: boolean;
```

Comment:
I started off using just `deprecated`, but decided to switch it to `deprecatedStateToken` because there is specific logic attached to these so the idea is that this flag would only be used on `state` tokens and not just on a general token that has been deprecated (which we don't currently have/support).